src/language-css/parser-postcss.js

```diff
@@ -261,15 +274,29 @@ function parse(text, parsers, opts) {
     ? opts.parser === "scss"
     : IS_POSSIBLY_SCSS.test(text);
 
+  let frontMatter = null;
+
+  try {
+    frontMatter = grayMatter(text);
+  } catch (originalError) {
```

Comment:
Why is this caught error called `originalError`?

Reply:
@lydell other name?

Reply:
`error` I guess